{"trial_id": "0080", "hyperparameters": {"space": [{"class_name": "Int", "config": {"name": "units_1", "default": null, "conditions": [], "min_value": 16, "max_value": 512, "step": 16, "sampling": "linear"}}, {"class_name": "Choice", "config": {"name": "bidir_1", "default": 1, "conditions": [], "values": [1, 0], "ordered": true}}, {"class_name": "Choice", "config": {"name": "use_second_lstm", "default": 1, "conditions": [], "values": [1, 0], "ordered": true}}, {"class_name": "Choice", "config": {"name": "use_attention", "default": 1, "conditions": [], "values": [1, 0], "ordered": true}}, {"class_name": "Int", "config": {"name": "units_2", "default": null, "conditions": [], "min_value": 16, "max_value": 512, "step": 16, "sampling": "linear"}}, {"class_name": "Choice", "config": {"name": "bidir_2", "default": 1, "conditions": [], "values": [1, 0], "ordered": true}}, {"class_name": "Int", "config": {"name": "attn_units", "default": null, "conditions": [], "min_value": 4, "max_value": 16, "step": 16, "sampling": "linear"}}, {"class_name": "Float", "config": {"name": "learning_rate", "default": 1e-05, "conditions": [], "min_value": 1e-05, "max_value": 0.01, "step": null, "sampling": "log"}}, {"class_name": "Int", "config": {"name": "reduce_lr_patience", "default": null, "conditions": [], "min_value": 2, "max_value": 5, "step": 1, "sampling": "linear"}}, {"class_name": "Choice", "config": {"name": "reduce_lr_factor", "default": 0.4, "conditions": [], "values": [0.4, 0.5, 0.6, 0.7], "ordered": true}}], "values": {"units_1": 496, "bidir_1": 0, "use_second_lstm": 0, "use_attention": 1, "units_2": 464, "bidir_2": 0, "attn_units": 4, "learning_rate": 0.0062144349964598534, "reduce_lr_patience": 4, "reduce_lr_factor": 0.4, "tuner/epochs": 2, "tuner/initial_epoch": 0, "tuner/bracket": 5, "tuner/round": 0}}, "metrics": {"metrics": {}}, "score": null, "best_step": 0, "status": "FAILED", "message": "Traceback (most recent call last):\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/base_tuner.py\", line 274, in _try_run_and_update_trial\n    self._run_and_update_trial(trial, *fit_args, **fit_kwargs)\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/base_tuner.py\", line 239, in _run_and_update_trial\n    results = self.run_trial(trial, *fit_args, **fit_kwargs)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/tuners/hyperband.py\", line 427, in run_trial\n    return super().run_trial(trial, *fit_args, **fit_kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/tuner.py\", line 314, in run_trial\n    obj_value = self._build_and_fit_model(trial, *args, **copied_kwargs)\n                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/tuner.py\", line 233, in _build_and_fit_model\n    results = self.hypermodel.fit(hp, model, *args, **kwargs)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/hypermodel.py\", line 149, in fit\n    return model.fit(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras/src/utils/traceback_utils.py\", line 122, in error_handler\n    raise e.with_traceback(filtered_tb) from None\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/tensorflow/python/eager/execute.py\", line 59, in quick_execute\n    except TypeError as e:\ntensorflow.python.framework.errors_impl.InternalError: Graph execution error:\n\nDetected at node gradient_tape/sequential_1/lstm_1/CudnnRNNBackpropV3 defined at (most recent call last):\n  File \"<frozen runpy>\", line 198, in _run_module_as_main\n\n  File \"<frozen runpy>\", line 88, in _run_code\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/ipykernel_launcher.py\", line 18, in <module>\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/traitlets/config/application.py\", line 1075, in launch_instance\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/ipykernel/kernelapp.py\", line 739, in start\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/tornado/platform/asyncio.py\", line 205, in start\n\n  File \"/home/jr/miniconda3/lib/python3.11/asyncio/base_events.py\", line 607, in run_forever\n\n  File \"/home/jr/miniconda3/lib/python3.11/asyncio/base_events.py\", line 1922, in _run_once\n\n  File \"/home/jr/miniconda3/lib/python3.11/asyncio/events.py\", line 80, in _run\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/ipykernel/kernelbase.py\", line 545, in dispatch_queue\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/ipykernel/kernelbase.py\", line 534, in process_one\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/ipykernel/kernelbase.py\", line 437, in dispatch_shell\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/ipykernel/ipkernel.py\", line 362, in execute_request\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/ipykernel/kernelbase.py\", line 778, in execute_request\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/ipykernel/ipkernel.py\", line 449, in do_execute\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/ipykernel/zmqshell.py\", line 549, in run_cell\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/IPython/core/interactiveshell.py\", line 3075, in run_cell\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/IPython/core/interactiveshell.py\", line 3130, in _run_cell\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/IPython/core/async_helpers.py\", line 128, in _pseudo_sync_runner\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/IPython/core/interactiveshell.py\", line 3334, in run_cell_async\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/IPython/core/interactiveshell.py\", line 3517, in run_ast_nodes\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/IPython/core/interactiveshell.py\", line 3577, in run_code\n\n  File \"/tmp/ipykernel_5204/1855840295.py\", line 130, in <module>\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/base_tuner.py\", line 234, in search\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/base_tuner.py\", line 274, in _try_run_and_update_trial\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/base_tuner.py\", line 239, in _run_and_update_trial\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/tuners/hyperband.py\", line 427, in run_trial\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/tuner.py\", line 314, in run_trial\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/tuner.py\", line 233, in _build_and_fit_model\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras_tuner/src/engine/hypermodel.py\", line 149, in fit\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras/src/utils/traceback_utils.py\", line 117, in error_handler\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras/src/backend/tensorflow/trainer.py\", line 371, in fit\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras/src/backend/tensorflow/trainer.py\", line 219, in function\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras/src/backend/tensorflow/trainer.py\", line 132, in multi_step_on_iterator\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras/src/backend/tensorflow/trainer.py\", line 113, in one_step_on_data\n\n  File \"/home/jr/miniconda3/lib/python3.11/site-packages/keras/src/backend/tensorflow/trainer.py\", line 77, in train_step\n\nFailed to call DoRnnBackward with model config: [rnn_mode, rnn_input_mode, rnn_direction_mode]: 2, 0, 0 , [num_layers, input_size, num_units, dir_count, max_seq_length, batch_size, cell_num_units]: [1, 5, 496, 1, 30, 64, 496] \n\t [[{{node gradient_tape/sequential_1/lstm_1/CudnnRNNBackpropV3}}]] [Op:__inference_multi_step_on_iterator_6123351]\n"}